lighthouse-core/config/default-config.js

```diff
@@ -424,6 +426,9 @@ const defaultConfig = {
     'best-practices-general': {
       title: str_(UIStrings.bestPracticesGeneralGroupTitle),
     },
+    'hidden': {
```

Comment:
can just `'hidden': '',`

and delete it from UIStrings.  
This string never shows up so we dont need it translated

Reply:
oh my bad.

yes as you pointed out offline i meant to say 

```
  'hidden': {
    title: '',
  },
```